### PR TITLE
Add function to restore a "trashed" user

### LINKED ISSUE #469: How to fully delete, or reactivate deleted, users?
We have deleted several users from Kimai, and they show up when we click "Show deleted users" as expected. However, we would like to fully delete these users, or reactivate them, so that we can use their usernames. How can we fully delete users, or reactivate them?

### PR DESCRIPTION
**Pull request by kevinpapst**
FIXES #469

This PR adds the functionality to restore a "trashed" user.
Currently we can only fully delete these users, now we can restore them as well (if they were deleted by accident).

Also done:
- Added a column to display the users global user in the users admin panel (currently we can only see the group memberships)
- Code style improvements "on the fly" ;-)
- Created a functions.php include for admin extensions, so the code is not 1:1 duplicated in init.php and processor.php
- Improved Kimai_View and using this class instead of Zend_View
- Improved some translations

### CODE REVIEW

File: extensions/ki_adminpanel/processor.php [hunk 1]
```diff
@@ -307,14 +287,25 @@ function ($group) {
 
         if (count($errors) == 0) {
             switch ($axValue) {
-                case 1 :
-                    // If the confirmation is returned the user gets the trash-flag.
+                // If the confirmation is returned the user is moved to trash
```
Comment (simonschaufi) [inline extensions/ki_adminpanel/processor.php]: wrong comment?

File: language/de.php [hunk 2]
```diff
@@ -337,8 +338,8 @@
         'customerWithSameName' => 'Benutzernamen dürfen nicht gleich lauten wie Kundennamen.',
         'sameGlobalRoleName' => 'Eine globale Rolle mit diesem Namen existiert bereits.',
         'sameMembershipRoleName' => 'Eine gruppenbezogene Rolle mit diesem Namen existiert bereits.',
-        'noActivitySelected' => 'Keine Aktivität ausgwählt.',
-        'noProjectSelected' => 'Kein Projekt ausgwählt.',
+        'noActivitySelected' => 'Keine Aktivität ausgewählt.',
```
Comment (simonschaufi) [inline language/de.php:341]: hey, that was schwäbisch :D

File: extensions/ki_adminpanel/processor.php (hunk 1 above)
Comment (kevinpapst) [inline extensions/ki_adminpanel/processor.php]: detective simon spotted a copy&paste mistake ;-) i'll push an update after you are done!

File: language/de.php (hunk 2 above)
Comment (kevinpapst) [inline language/de.php:341]: :+1:

**Comment by kevinpapst** [comment]
Updated

**Pull request merged**